file-formats/scp1/type/zif_aff_scp1_v1.intf.abap

```diff
@@ -250,7 +228,11 @@ INTERFACE zif_aff_scp1_v1
     BEGIN OF ty_sub_bcset,
       "! <p class="shorttext">Name</p>
       "! Name of the BC Set
-      name TYPE c LENGTH 32,
+      name     TYPE c LENGTH 32,
+      "! <p class="shorttext">Position</p>
+      "! Position of BC Set in Hierarchical BC Set
+      "! $maxLength 10
```

Comment:
```suggestion
      "! $maximum 10
```

Reply:
Changed

Reply:
maximum 10 means the highest value "position_in_hierarchy" can have is 10.
maxLength is only working for character-like fields (f.ex. n or c). You can read more about it here https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#type-mapping

maxLength 10 would not have an effect on abap integers because an abap integer can have at most 10 digits (maximum is 2147483647)

Reply:
changed to $maximum 10